### ISSUE TITLE
monster-presents-insomniac-episode-05 - Rev

### ISSUE THREAD

**Issue by tg-z** [tg-z/web-clip#206]
The views and opinions expressed in this podcast are solely those of the authors and participants and do not necessarily represent those of iHeartMedia, Stuff Media, or it's employees. Listener discretion is advised. From iHeartRadio and Tenderfoot TV, Monster presents, "Insomniac."

I'd like to talk about a bad neighbor, and I'm not talking about just any ordinary bad neighbor, like somebody who allows their dog to bark both day and night or blocks your driveway with your car. No, I'm talking about a creepy, antisocial neighbor that turned out to be far worse than anyone ever expected. He was a predator and he selected the girl next door as his prey. As far as neighbors go, it doesn't get much worse. A story like the one you're about to hear sparks fear in all of us. You never know who might move in your neighborhood and you certainly have no idea what they're up to or capable of behind closed doors. I'm Scott Benjamin and everything I'm about to tell you is real. This is Insomniac.

A killer was living among the students at Barrister's Hall Apartments in Macon, Georgia. It was Thursday morning, June 30th, 2011, and it was garbage collection day. You make sure you get your trash to the curbside at the usual time and when the big truck rolls away after tossing your bags in the back, you're relieved that you got rid of that week's waste. At least, that's what usually happens, but this particularly morning was a little different. Not only were the garbage men running a few minutes late, but just moments before the truck arrived, the bins were inadvertently blocked by a pair of Macon Georgia police cars that had pulled into the parking lot. The detectives were there to investigate a missing person's report called in early that morning. Lauren Giddings, a 27-year-old recent graduate of Mercer Law School, hadn't been by her friends or family since evening of June 25th, five days prior. The police weren't intentionally blocking the trash bins. It was only by chance that they had parked where they did. When the garbage truck arrived moments later, the driver realized the containers were unreachable, so he simply smiled, waved at the detectives, and kept on driving to his next stop.

Moments later, the authorities knocked on the door of Stephen McDaniel, Lauren's next door neighbor in the A unit apartment building. He was asked a few routine questions about his missing neighbor, then left alone as the police went about their work. It was several hours later, that same afternoon, when a very talkative Stephen McDaniel agreed to appear on the evening newscast during a curbside interview with a local television news station, WGXATV. Just a couple of minutes in, he was blindsided by the news. It changed his demeanor completely. He was visibly affected by the information the reporter was sharing with him. It was at that moment, that very public moment, that Stephen McDaniel knew he was in trouble, and thanks to the video crew there to document it, the entire city of Macon, Georgia knew it too.

Today's episode is a little bit out of the ordinary in that we're going to take a look at a killer who only claimed one victim. Now, that's not in any way to downplay the fact that this person took a human life. However, I think you'll find the way today's case unfolds is just as interesting and horrific as any other case involving a serial killer. Some of the details are so unusual that the story almost seems like fiction, but it's all real and it really did happen this way. Now, unless you're from the southeastern part of the United States, or unless you happen to be really up on your true crime, there's a good chance you've never even heard of this case. There's really no mystery about who did the killing. There are no red herring suspects for us to introduce, no other leads to follow, and, in the end, there will be no doubt remaining that the real killer is paying the price.

You see, right from the beginning, it was evident that Stephen McDaniel was the primary person of interest in the murder of his neighbor, Lauren Giddings. You'll understand why the detectives were almost immediately onto Stephen as we sort through the details. In fact, despite his best effort to pull off what he considered to be the perfect murder, Stephen McDaniel made it fairly easy for the detectives right from the beginning. He made several mistakes that he simply couldn't hide and, like most other narcissistic killers who are captured, he quickly came to the crushing realization that he wasn't necessarily smarter than everybody else, including his pursuers.

As we lay out the details of today's case, I'd like you to keep this at the forefront of your thoughts. The authorities have a theory that Stephen McDaniel was a serial killer in the making, but because he wasn't very good at it, they just happened to catch him after his first murder. Had they delayed in his capture, it's likely there would have been more victims. As you listen to the rest of today's episode, take note of some of Stephen's peculiar behaviors and ask yourself if you agree. We'll start in a town that sits about 84 miles southeast of Atlanta, one that goes by the nickname, "The Heart of Georgia," the city of Macon.

Among the newest graduates of Mercer Law School in May of 2011 were two students that knew of each other, yet, as it turns out, didn't know each other very well. For about three years the two lived right next door to one another. They even shared a common stairway to their second floor residences at the Barrister's Hall Apartment Complex, right across the street from the college campus. Other than that, the two had nothing more in common. Lauren Teresa Giddings was a 27-year-old, blonde-haired native of Laurel, Maryland. She was known by her classmates, friends, and family as athletic, cheerful, outgoing, and social. She had ambitions of putting her law degree to good use in her new career as a lawyer in the nearby city of Atlanta. After her graduation, all that remained in her way was the Georgia Bar Exam. Her plans, like many other graduating students in her class, were to stay in Macon in order to study for the bar exam throughout the months of May and June. Lauren had a steady boyfriend, a corporate lawyer in Atlanta named David VandeVere. He and Lauren had been together since 2007 when Lauren accepted an internship as a project assistant at the law office where David worked in downtown Atlanta. Lauren never knew it, but David was planning to propose to her after graduation on a surprise trip to Bermuda later that fall.

In stark contrast to Lauren's character was her next-door neighbor, Stephen Mark McDaniel. He was 25 years old and nothing like his friendly, sociable neighbor, Lauren. Stephen grew up in the town of Lilburn, Georgia, a northeast suburb of the city of Atlanta. He was known on campus as a sort of odd character. He was generally antisocial, withdrawn, and more than a little eccentric in his behavior and interactions with others. He was quirky, but according to those who knew him, he was also very smart. Stephen's second-floor apartment was a place of retreat for him, somewhere he could go to be alone. He was essentially a 25-year-old recluse, staying inside most of the time with his video games, his pornography, his guns, and his swords. Stephen was a prepper, collecting food and supplies in the way a survivalist might. He also collected empty containers for some reason, soda bottles, large plastic jugs, and even the cardboard centers of toilet paper rolls. He owned an older car, one that he rarely used, a 1997 Geo Prism. It was driven so infrequently that cobwebs would form between the wheels and the ground.

From the beginning, Stephen had been interested in Lauren, but it was a one-way street. He had asked her out at least a couple of times, only to be rejected on all occasions. After all, she did have a boyfriend, but even if she wasn't in a relationship, Lauren would not have accepted Stephen's advances. She thought of him as the creepy next-door neighbor who rarely went outside, but of course, she had enough social grace to never outright say that to Stephen. We should also note that Lauren was never known to be afraid of Stephen McDaniel, at least she never mentioned to anyone that she was frightened by her strange neighbor. What she didn't know, however, definitely would have scared her. Because he couldn't have her, Stephen began to obsess over his neighbor and that led to some secretive and troubling behavior in the apartment next door to Lauren's.

Early in the case, detectives were given access to Stephen's Internet browsing history and it revealed some disturbing patterns. On April 28th, 2011, about two months before he killed Lauren, Stephen typed the Google search phrase, "Nude Lauren Giddings." He also viewed Lauren's Twitter feed. Later he did several back-to-back searches using variations of the word "Molest sleeping girl." The very next day, he searched for the phrase, "Choked unconscious, how long wake up." Going back as far as May first he searched for ways to escape prison. Throughout the months of May and June, his Internet browsing history showed a growing obsession with Lauren Giddings. Within the same timeframe, he visited sites to search for nude photos of celebrities, advertisements for local escorts, dating sites, reading erotic fiction, and viewing pornography. Using the screen name SOL, which he stated stands for Son of Liberty, he wrote graphic sexual posts online describing torture and violence towards women. He looked at guns and sex toys as well.

By June 3rd, 2011, he was looking at Lauren's Amazon.com wishlist. On June 7th, 2011, he was on a photo sharing website looking for Lauren Gidding's account. The next day, June 8th, 2011, McDaniel looked at Lauren's LinkedIn networking page and looked at her Facebook page as well. None of these searches are illegal, of course, but they paint an increasingly dark image of Stephen McDaniel's character in the months prior to Lauren's murder.

Lauren Giddings and Stephen McDaniel. The two didn't know it yet, but they were about to make headlines, not because they were young, superstar lawyers, as they had both hoped to soon be, but because of the awful situation that was to unfold in late June, 2011.

A little more than one month after graduation, Lauren spent her final afternoon poolside at Healy Point Country Club in River North, and then around 5:48 P.M. she used her credit card to buy dinner at a Zaxby's drive-through on her way home. Her outstretched arm was captured on video as she reached for her order from the drive-through window. That was the last outside contact anyone had with Lauren before she was attacked and murdered in the early morning hours of Sunday, June 26th, 2011. No one, with the exception of her killer, would know where Lauren was for the next five days.

You might wonder how a socially connected person like Lauren might go unmissed for nearly a week. Well, her friends and family knew that she was preparing for the Georgia Bar Exam and gave her the privacy she would need to study for her test. Everyone close to her would have understood why she was secluded in her apartment, locked away and temporarily unreachable. They didn't want to disturb her.

On the afternoon of June 25th, the evening before Stephen McDaniel snuck into Lauren's apartment to kill her, he looked at her Facebook page once again. Then, just before her murder, in the early morning hours of June 26, he searched multiple times for ways to defeat the door-jamming burglar bar that Lauren Giddings used to keep out intruders. But how did Stephen know what was behind her closed door inside her apartment?

The past several years have been difficult to get through. I lost my aunt in 2012, then my grandmother died in 2014, my grandfather in 2015, and then my own mother in 2017. Along the way, I also lost my first dog. He had been with us for 16 years. I loved him dearly. We also lost a bird that was with us for almost 14 years. The family pets, well, I'm sure that most of you can relate to that. They become another member of the family and you're just as sad to see them go as anyone else. It was sometime during the middle of all of this that I started having trouble sleeping. I thought I was mentally prepared, as much as possible anyway, to let those people in my life go. The death of someone very old or someone with a fatal diagnosis should be expected sooner or later, but that doesn't make it any easier and I never thought their deaths would affect me quite as much as they have over the past several years.

The pain is fading, but not fast enough. All of these people, my aunt, my grandparents, and my mother, still occupy my thoughts most days and sometimes they're in my dreams, both good and bad. I'm telling you this because it's not just working on this podcast that's been the cause of my nightmares. It's also been these events of my personal life, but being able to put it all out there, as bad as it is, is helping me find my way out.

Just before Stephen McDaniel's case was to go to trial, it was discovered that, a few hours before he killed Lauren Giddings, between 9:00 PM and 12:30 AM, Stephen McDaniel was sneaking around outside of Lauren's second-story living room window. He was watching Lauren. He was looking through her blinds by using a video camera duct taped to the end of a six foot wooden stick as he stood at ground level. As he scanned the inside of Lauren's apartment with the camera, he paid particular attention to her front door and how it was secured with a burglar bar. He was looking for a way to defeat the device that stretched from the floor to the doorknob. The video clips showing the inside of Lauren's apartment were deleted from Stephen's camera in the days that followed her death, but the erased files were recovered years later by the FBI as the evidence continued to grow against him and he was headed to trial.

Things were looking bad for Stephen, real bad. Not only did the prosecutors now have the deleted stalking videos, they also had child pornography images that had been stored on Stephen's computer, adding 30 counts of sexual exploitation of children to the charges against him. These were unrelated to the murder, but he was now looking at additional prison time, five to 20 years for each of the 30 charges.

On June 30th, 2011, a nearly unbelievable series of events took place throughout the day, along with Stephen's unexpected and shocking confession late in the day. You'll get to hear from Stephen McDaniel himself as he's interviewed just outside of his apartment the moment he learns a body was discovered, next time on Insomniac.

Insomniac is a production of iHeartRadio and Tenderfoot TV, written and hosted by Scott Benjamin and produced by Miranda Hawkins, Alex Williams, Matt Friedrich, and Josh Thane. Music composed by Makeup and Vanity set and cover art by Trevor \[Iler 00:18:38\]. Follow on Twitter and Facebook at Insomniac Pod, on Instagram at Insomniacpodcast and at our website, insomniacpodcast.com. For more podcasts from iHeartRadio, visit the iHeartRadio app, Apple Podcasts, or wherever you listen to your favorite shows.
